### PR TITLE
Rakesh | MOBN-1411 | Added min width for fieldset

### PR DESCRIPTION
**Pull request by rakesh-1010**
Rakesh | MOBN-1411 | Added multiline and max-row for name, shortcode in dictionaries and sources details page
Rakesh | MOBN-1411 | Wrap whiteSpace for supported locales dropdown

# JIRA TICKET NAME:
[JIRA ticket name](https://issues.openmrs.org/browse/<ticket-id>)

# Summary:

### CODE REVIEW

File: src/apps/dictionaries/components/DictionaryForm.tsx [hunk 1]
```diff
@@ -71,6 +71,20 @@ const useStyles = makeStyles({
   }
 });
 
+const supportedLocalesLabel = (values: any) => {
```
Comment (kirity-mahanty) [inline src/apps/dictionaries/components/DictionaryForm.tsx:74]: For Source details also, we have the supported locale field. Can we make it common and use for both dictionaries and sources.

File: src/apps/dictionaries/components/ReleasedVersions.tsx [hunk 2]
```diff
@@ -105,7 +105,7 @@ const ReleasedVersions: React.FC<Props> = ({
                 {versionsToDisplay.map((row: APIDictionaryVersion) => (
                   <TableRow key={row.id}>
                     <TableCell>{row.id}</TableCell>
-                    <TableCell>{row.description || "None"}</TableCell>
+                    <TableCell style={{wordBreak: 'break-all'}}>{row.description || "None"}</TableCell>
```
Comment (kirity-mahanty) [inline src/apps/dictionaries/components/ReleasedVersions.tsx:108]: Don't we need the same style for row.id as well? version is also can take a large string value.

File: src/apps/dictionaries/pages/ViewDictionaryPage.tsx [hunk 3]
```diff
@@ -99,7 +99,7 @@ const ViewDictionaryPage: React.FC<Props> = ({
     >
       <Grid id="viewDictionaryPage" item xs={5} component="div">
         <Paper className="fieldsetParent">
-          <fieldset>
+          <fieldset style={{minWidth: "0"}} >
```
Comment (kirity-mahanty) [inline src/apps/dictionaries/pages/ViewDictionaryPage.tsx:102]: Did not understand the usage of this style. We will discuss this.

Comment (rakesh-1010) [reply src/apps/dictionaries/pages/ViewDictionaryPage.tsx:102]: @kirity-mahanty fieldset is the black border outline, this css property keeps the fieldset from overflowing out of page boundary

File: src/apps/dictionaries/components/ReleasedVersions.tsx (hunk 2 above)
Comment (rakesh-1010) [reply src/apps/dictionaries/components/ReleasedVersions.tsx:108]: @kirity-mahanty added

File: src/apps/dictionaries/components/DictionaryForm.tsx (hunk 1 above)
Comment (rakesh-1010) [reply src/apps/dictionaries/components/DictionaryForm.tsx:74]: @kirity-mahanty Added the same function for sources as well, but making it common doesn't seem reasonable as this function is returning a jsx element. Will have to make it as a component., as it is returning a group of material ui component

File: src/apps/dictionaries/components/ReleasedVersions.tsx (hunk 2 above)
Comment (kirity-mahanty) [reply src/apps/dictionaries/components/ReleasedVersions.tsx:108]: thanks.

File: src/apps/dictionaries/pages/ViewDictionaryPage.tsx (hunk 3 above)
Comment (kirity-mahanty) [reply src/apps/dictionaries/pages/ViewDictionaryPage.tsx:102]: ok.

**Pull request merged**